<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loadcell_second</class>
 <widget class="QMainWindow" name="loadcell_second">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../AEL form/AEL-removebg-preview.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>称重传感器校准模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请输入砝码或已知的标准重量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>159</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>下一步</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_home">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>主页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>159</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上一步</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>5kg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10kg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15kg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20kg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30kg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50kg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100kg</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
